src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Response.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Response
+{
+    public required int BibleId { get; set; }
+    public required string BibleName { get; set; }
+    public required string BibleAbbreviation { get; set; }
+    public required string BookName { get; set; }
+    public required string BookCode { get; set; }
+    public required IReadOnlyList<ResponseChapter> Chapters { get; set; }
+}
+
+public class ResponseChapter
+{
+    public required int Number { get; set; }
+    public required ResponseChapterAudio? Audio { get; set; }
```

Comment:
Note that specifying `false` for `ShouldReturnAudioData` will still result in this `Audio` property being returned in the response JSON, though it will have a `null` (same for the verse's `audioTimestamp`).  If we are earnest about saving bandwidth then there are two options to improve this:
1. Use `[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]` on these properties.
2. Adjust global JSON serialization settings to omit `null` values for all responses.

I'm in favor of (1) here because the consumer didn't explicitly ask for audio data.  That said, removing the audio properties altogether does make the fact that the route returns audio less discoverable.

Thoughts?

Reply:
I went ahead and did (1) and confirmed that it works as expected: [commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/2573f29a855ad8a628d9f201ed7711002c56dd76).

Note that if `shouldReturnAudioData` is `true` but there is no audio data for the given Bible/book/chapter/verse then we _also_ won't include the JSON properties.  This may be unexpected for consumers.

I'd argue that this is fine because the Bibles List route added in this PR states if Bibles have audio and consumers of JSON can generally handle missing JSON properties by supplying `null`.

If this isn't acceptable and we still want the bandwidth improvements then I see three potential ways to solve this:
1. Create a [custom serialization contract](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/custom-contracts) that only omits `null` properties for the audio types based upon another (non-serialized) property on the object.  AKA add `shouldReturnAudioData` to the response object, don't serialize it, and use it to conditionally exclude `null` audio properties.
2. Use two different response objects (probably through inheritance).  Return one that has audio properties if `shouldReturnAudioData == true` and the other that doesn't have audio properties if `shouldReturnAudioData == false`.
3. Use a separate route for audio data.

Without objection I'll leave this as-is (with my commit at the top of this comment in place).